src/containers/WatchList/DeleteMovie/container.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal } from 'antd'
+import { connect } from 'react-redux'
+import { openNotification } from 'Utils'
+
+import { removeMovieErrorsSelector } from 'Store/watchList/selectors'
+import {
+  removeMovie as removeMovieAction,
+  cleanupRemoveMovieErrors as cleanupRemoveMovieErrorsAction
+} from 'Store/watchList/actions'
+import DeleteMovieComponent from './component'
+
+const DeleteMovie = ({
+  movieId,
+  title,
+  removeMovie,
+  removeMovieErrors,
+  cleanupRemoveMovieErrors
+}) => {
+  const config = {
+    message: 'Error',
+    description: `Cant remove "${title}" from Watchlist`,
+    type: 'error',
+    onClose() { cleanupRemoveMovieErrors(movieId) }
+  }
+  const error = removeMovieErrors.includes(movieId) && openNotification(config)
+  return (
+    <DeleteMovieComponent
+      onClick={() => Modal.confirm({
+        title: 'Do you want to delete movie from watchlist?',
+        onOk() { removeMovie(movieId, title) }
+      })}
```

Comment:
extract this to the method like `handleDeleteMovie`

Reply:
✅